pipeline.yml

```diff
@@ -880,10 +879,9 @@ jobs:
       resource: masterbosh-state
   - task: decrypt-private-key
     file: pipeline-tasks/decrypt.yml
-    config:
-      inputs:
-      - name: pipeline-tasks
-      - name: common-masterbosh
+    input_mapping:
+    - name: pipeline-tasks
```

Comment:
I think that notice has been in place for a year, and I'm guessing it's not going to actually get deprecated until https://github.com/concourse/concourse/issues/923 is resolved. If we do want to avoid the warning today, we'd need to add a second input to the decrypt task in pipeline-tasks, or define the whole config inline.